### PR TITLE
Fixes #501: Adding lazy loading image in media wall

### LINKED ISSUE #501: Introducing lazy loading image module in Media wall
Idea is to introduce lazy load image module in media wall.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
-  Adding lazy loading image in media wall

**Test Link**
https://achint08.github.io/loklak_search/

**Closes #501 **

### CODE REVIEW

**Comment by Achint08** [comment]
Please review @djmgit @hemantjadon  @kavithaenair @SKrPl @vibhcool @singhpratyush

**Comment by Achint08** [comment]
@singhpratyush These are because of images received from the server as `https://www.instagram.com/p/BXG_ZjljRus/` which are particularly not images but link to pages.

File: src/app/media-wall/media-wall-card/media-wall-card.component.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ div.masonry-content {
 			margin: 0 8px 1px 0;
 			overflow: hidden;
 
-			img {
+			/deep/ img {
 				float: left;
```
Comment (hemantjadon) [inline src/app/media-wall/media-wall-card/media-wall-card.component.scss]: Why using floats ?

File: src/app/media-wall/media-wall-card/media-wall-card.component.scss [hunk 2]
```diff
@@ -23,7 +23,7 @@ div.masonry-content {
 			margin: 0 8px 1px 0;
 			overflow: hidden;
 
-			img {
+			/deep/ img {
```
Comment (hemantjadon) [inline src/app/media-wall/media-wall-card/media-wall-card.component.scss]: It should be `app-lazy-img /deep/ img`

Comment (Achint08) [inline src/app/media-wall/media-wall-card/media-wall-card.component.scss]: Sure.

**Comment by Achint08** [comment]
@hemantjadon Done! 👍

**Comment by mariobehling** [comment]
This feature is ok, but when will we have a smooth update of new tweets on the top and other tweets moving to the bottom like in other standard walls, where tweets are moving in instead of substituting the top? Please help to get the basics right.

**Pull request merged**